editor.planx.uk/src/@planx/components/MapAndLabel/Public.tsx

```diff
@@ -17,6 +19,18 @@ function MapAndLabelComponent(props: Props) {
         policyRef={props.policyRef}
         howMeasured={props.howMeasured}
       />
+      <MapContainer environment="standalone">
+        <my-map
+          drawMode
+          drawPointer="crosshair"
+          zoom={20}
+          drawFillColor={alpha(props.drawColour, 0.1)}
+          drawColor={props.drawColour}
+          drawPointColor={props.drawColour}
```

Comment:
Really like how you're using a single draw color input to cross-populate all necesary style props here - also feels like good inspiration to think about deprecating some of those redundant map props & simplifying at the source if we can :+1: